contracts/HubPool.sol

```diff
@@ -90,7 +114,7 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
         uint256 lpTokensBurnt,
         address indexed liquidityProvider
     );
-    event WhitelistRoute(address originToken, uint256 destinationChainId, address destinationToken);
```

Comment:
I changed the ordering around for any method that takes in a chainId to be the _first_ prop in the event/function ect.

Reply:
nit: index by originToken and destinationChainId?